extensions/amp-twitter/1.0/component.js

```diff
@@ -11,8 +11,8 @@ const FULL_HEIGHT = '100%';
 const MATCHES_MESSAGING_ORIGIN = () => true;
 
 /**
- * @param {!TwitterDef.Props} props
- * @param {{current: (!TwitterDef.Api|null)}} ref
+ * @param {!BentoTwitterDef.Props} props
+ * @param {{current: (!BentoTwitterDef.Api|null)}} ref
  * @return {PreactDef.Renderable}
  */
 function TwitterWithRef(
```

Comment:
Should this function also be renamed `BentoTwitterWithRef` 
(referencing the FitText change the function was also renamed)

Reply:
`BentoFitText` is exported while this is not, therefore I did not rename this.

Reply:
For consistency I'd encourage naming this `BentoTwitterWithRef` because that's what it is - `BentoTwitter` component that takes a `ref`